Terraform/legos/terraform_exec_command/terraform_exec_command.py

```diff
@@ -46,13 +46,14 @@ def terraform_exec_command(handle, repo, dir_path, command) -> str:
         :rtype: Str Output of the command .
     """
     assert(command.startswith("terraform"))
-
     print(f'WARNING: Please note terraform apply and terraform destroy will be run with -auto-approve for non-interactive run')
 
     output = ''
     # sanitize inputs that have come from validate 
 
     try:
+        if dir_path == '':
+            dir_path = './'
```

Comment:
why are we doing this hidden way, why not make the default value as ./ in the schema